src/app/pages/installation/_linux_designer.twig

```diff
@@ -0,0 +1,30 @@
+<p>
```

Comment:
just for your info: this file must be true for all versions below:
- LTS
- LE
- Nightly
- Dev
- Nightly-8
- Sprint
- Nighlty-7

So you may mention special versions of Axon Ivy (e.g. since 8.0 you need to ..)
but it looks good ..

Reply:
yet I think the guide fits for versions ... later I guess we will hide the Ubuntu section once we no longer depend upon X.org